downstream/modules/eda/proc-modifying-memory-after-install.adoc

```diff
@@ -0,0 +1,11 @@
+[id="proc-modifying-memory-after-install"]
+
+= Modifying the default memory limit for each rulebook activation after installation
+
+[role="_abstract"]
+By default, each rulebook activation container has a 200mb memory limit. You can modify this default value after installation by using the following procedure.
+
+.Procedure
+. Navigate to the environment file at `/etc/ansible-automation-platform/eda`.
+. Modify the default container memory limit. For example, `EDA_PODMAN_MEM_LIMIT = 300m`.
```

Comment:
@jainnikhil30 Does 300m also need to be in '300m' single quotes?

Reply:
Oh yes, good catch! it needs to be single quoted yes.